sr_description/hand/xacro/finger/proximal/proximal.gazebo.xacro

```diff
@@ -1,24 +1,24 @@
 <?xml version="1.0"?>
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
-  <xacro:macro name="proximal_gazebo" params="prefix link_prefix">
-
-  <!-- prefix can be anything but usually is "r_" or  "l_" for right and left hands distinction -->
-  <gazebo reference="${prefix}${link_prefix}proximal">
-      <sensor type="contact" name="${prefix}${link_prefix}proximal_contact">
-        <always_on>1</always_on>
-        <update_rate>1000.0</update_rate>
-        <contact>
-          <collision>${prefix}${link_prefix}proximal_collision</collision>
-          <topic>${prefix}${link_prefix}proximal_collision</topic>
-        </contact>
-        <plugin name="${prefix}${link_prefix}proximal_gazebo_ros_bumper_controller" filename="libgazebo_ros_bumper.so">
-          <alwaysOn>true</alwaysOn>
-          <frameName>${prefix}${link_prefix}proximal</frameName>
-          <updateRate>100.0</updateRate>
-          <bumperTopicName>contacts/${prefix}${link_prefix}/proximal</bumperTopicName>
-        </plugin>
-      </sensor>
-        
+  <xacro:macro name="proximal_gazebo" params="prefix:=^ link_prefix:=^ prox_sensor:=^|std">
```

Comment:
```suggestion
  <xacro:macro name="proximal_gazebo" params="prefix:=^ link_prefix:=^ prox_sensor:=^|'std'">
```

Reply:
As this is not in an eval context, i.e. `${}`, we don't need quotes to specify a simple string.